check-kernel-config

```diff
@@ -270,6 +280,7 @@ CONFIG_SECURITY_APPARMOR_BOOTPARAM_VALUE=1
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_DEFAULT_IOSCHED=\"deadline\"
 CONFIG_EVM_HMAC_VERSION=2
+CONFIG_ANDROID_BINDER_DEVICES=\"binder,hwbinder,vndbinder,anbox-binder,anbox-hwbinder,anbox-vndbinder\"
```

Comment:
Does this line work correctly with -w option?

Reply:
It should, see "deadline" above. I can test though again.